src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryArgumentCollection.java

```diff
@@ -198,8 +199,8 @@
         public int assemblyImpreciseEvidenceOverlapUncertainty = DEFAULT_ASSEMBLED_IMPRECISE_EVIDENCE_OVERLAP_UNCERTAINTY;
 
         @Argument(doc = "Number of pieces of imprecise evidence necessary to call a variant in the absence of an assembled breakpoint.",
-                fullName = "imprecise-evidence-variant-calling-threshold")
-        public int impreciseEvidenceVariantCallingThreshold = DEFAULT_IMPRECISE_EVIDENCE_VARIANT_CALLING_THRESHOLD;
+                fullName = "imprecise-variant-evidence-threshold")
```

Comment:
Alternatively, can we still call such variants, and filter out the really large ones? Is performance a concern if we go this way?